### ISSUE TITLE
DMWM-WMCore-pylintpy3 jenkins job is failing with python exceptions

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#11049]
**Impact of the bug**
WMCore Jenkins

**Describe the bug**
It's been a few days that I do not see any meaningful Py3 pylint report, looking into the latest run for that job in Jenkins:
https://cmssdt.cern.ch/dmwm-jenkins/job/DMWM-WMCore-pylintpy3/lastBuild/console

gives me an exception like [1]

**How to reproduce it**
Something like:
```
docker run -v ~/Pycharm/WMCore:/home/dmwm/wmcore_unittest/WMCore -it --entrypoint /bin/bash gitlab-registry.cern.ch/cmsdocks/dmwm:wmcorepy3_tests

then
pylint
```

**Expected behavior**
Python3 pylint checks should be able to complete and the report used to evaluate our baseline code and PR changes.

**Additional context and error message**
[1]
```
Printing Pylint version
Traceback (most recent call last):
  File "/home/dmwm/unittestdeploy/wmagent/2.0.0.pre3/sw/slc7_amd64_gcc630/external/py3-setuptools/51.3.3/lib/python3.8/site-packages/pkg_resources/__init__.py", line 568, in _build_master
    ws.require(__requires__)
  File "/home/dmwm/unittestdeploy/wmagent/2.0.0.pre3/sw/slc7_amd64_gcc630/external/py3-setuptools/51.3.3/lib/python3.8/site-packages/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/dmwm/unittestdeploy/wmagent/2.0.0.pre3/sw/slc7_amd64_gcc630/external/py3-setuptools/51.3.3/lib/python3.8/site-packages/pkg_resources/__init__.py", line 777, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (astroid 2.4.2 (/home/dmwm/unittestdeploy/wmagent/2.0.0.pre3/sw/slc7_amd64_gcc630/external/py3-astroid/2.4.2-comp/lib/python3.8/site-packages), Requirement.parse('astroid==2.5.0'), {'pylint'})
```

**Comment by amaltaro** [comment]
This is the problem:
``` 
pkg_resources.DistributionNotFound: The 'astroid==2.5.0' distribution was not found and is required by pylint
```

**Comment by amaltaro** [comment]
This has been fixed with https://github.com/cms-sw/cmsdist/pull/7713
and a new wmcorepy3 Docker container has been made available (with WMCore release 2.0.1)
